src/documents/ChangeApplierAPI.md

```diff
@@ -446,29 +453,21 @@ These are not recommended for typical users of the framework, and are not stable
 
 ### Instantiating a ChangeApplier ###
 
-Instantiating a ChangeApplier manually is possible but is not recommended for general use. In particular, manual instantiation of a ChangeApplier will not allow access to the powerful [Model Relay](ModelRelay.md) framework features, which require the ChangeApplier to be properly situated in an IoC component tree. However, it's possible to imagine some cases where authors of "superframeworks" might find this facility useful. In Infusion 1.5 there are several variants for construction of ChangeAppliers since the implementation is in transition from the old non-relay-aware applier to the new model relay implementation.
-
-```javascript
-var applier = fluid.makeChangeApplier(model, options); // currently produces an "old-style" applier
-
-    OR
-var applier = fluid.makeHolderChangeApplier(holder, options); // currently produces an "old-style" applier
-    OR
-var applier = fluid.makeNewChangeApplier(holder, options); // produces a new "relay-aware" applier
-```
-
-In every case the `options` argument is optional. Where the parameter `model` appears it directly represents the model to be managed by the applier. Where parameter `holder` appears, it represents instead an object which is expected _to be a container for the model_ where the model itself is held in this object at the member named `model`. In framework use this argument will actually hold the owning component itself, although the applier makes no use of any properties of this object other than the one held at `model`.
+Instantiating a ChangeApplier manually is no longer recommended in current versions of the framework. Its implementation is tightly bound into its location in 
+an IoC component tree and should be constructed by the IoC system itself.
 
 Under the old ChangeApplier semantics, the `model` handle was closed over and remained constant for the lifetime of the applier. The framework contained utilities such a `fluid.model.copyModel` which assisted the user in working with a model with changing contents whilst keeping its base reference constant. Under the new ChangeApplier semantics, the `model` base reference is not stable and in fact starts by holding the value `undefined` as every model component initialises.
 
 ### Operating transactions manually ###
 
-Similarly to manual construction of a ChangeApplier, this is not recommended for normal users of the framework. It may be appropriate for advanced users of the framework who are building higher-level frameworks layered over it.
+A user may be interested in economising on notifications to model updates - by batching these up into a single transaction, there will
+just be a single notification of each listener which is impacted around the model skeleton. This facility is unstable and will be 
```

Comment:
It's ok to use but it is a very annoying and cumbersome API - noone would use it in its current form unless the need were really pressing. Also, it has no what you could call "declarative" or "externalisable" equivalent - that is, it is not possible to describe the required transactional structure from environments outside the process, e.g. by means of the NEXUS. Consult https://wiki.fluidproject.org/display/fluid/New+New+Notes+on+the+ChangeApplier if you are very interested in the details :)
In short, it is both "ok to use for the time being" (since there is no alternative) as well as being an unstable API.